packages/thirdweb/src/chains/chain-definitions/polygon.ts

```diff
@@ -6,7 +6,7 @@ import { defineChain } from "../utils.js";
 export const polygon = /*@__PURE__*/ defineChain({
   id: 137,
   name: "Polygon",
-  nativeCurrency: { name: "MATIC", symbol: "MATIC", decimals: 18 },
+  nativeCurrency: { name: "POL", symbol: "POL", decimals: 18 },
```

Comment:
sneaking it in huh xD

Reply:
this prob needs a changeset on its own, would separate it tbh

Reply:
eh its already changed on chain api I just forgot it here